src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector/nuxeo-aspera-upgrade-notes.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Nuxeo Aspera Upgrade Notes
+description: Upgrade notes for Nuxeo Aspera.
+tree_item_index: 200
+review:
+  comment: ''
+  date: '2020-09-10'
+  status: ok
+toc: true
+---
+
+
+{{! excerpt}}
+This chapter highlights some major information about upgrade from Nuxeo Aspera Connector 2.x to Nuxeo Aspera Connector 3.0.x.
+For a listing of improvements and functional enhancements in version 3.0.x, please see the [release notes]() {{! /excerpt}}
+
+{{#> callout type='warning'}}
+Because of the changes to parameters in the nuxeo.conf file, this release is labeled as a major release.  Transfers that exist in your system **that were started/created with a previous version will not work with this release and attempts to complete them will fail with this realse**.
+{{/callout}}
+
+## Installation and Configuration
+
+### Changed/modified Parameters
+
+<div class="table-scroll">
+<table class="hover">
+<tbody>
+<tr>
+<th colspan="1">Parameter</th>
+<th colspan="1">Modification</th>
+<th colspan="1">Reference</th>
+</tr>
+<tr>
+<td colspan="1">`aspera.access.key.id`
+`aspera.access.key.secret`
+`aspera.download.access.key.id`
+`aspera.download.access.key.secret`</td>
+<td colspan="1">Fixed existing typo for `acess` in these parameters</td>
+<td colspan="1">[NXP-28399](https://jira.nuxeo.com/browse/NXP-28399)</td>
+</tr>
+<tr>
+<td colspan="1">`nuxeo.s3storage.useDirectUpload = true`</td>
+<td colspan="1">Value no longer necessary for the Nuxeo Aspera Connector (keep if used for other Nuxeo features) </td>
+<td colspan="1">[NXP-29548](https://jira.nuxeo.com/browse/NXP-29548)</td>
+</tr>
+<td colspan="1">`nuxeo.s3storage.transient.roleArn`</td>
+<td colspan="1">Value no longer necessary for the Nuxeo Aspera Connector (keep if used for other Nuxeo features)</td>
+<td colspan="1">[NXP-29548](https://jira.nuxeo.com/browse/NXP-29548)</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+### Requirements
+- Aspera Desktop Client - [latest version]
+- Nuxeo Server LTS 2019 (10.10, HF 31, at least) with access to AWS S3 Storage along with the [Amazon S3 Online Storage plugin](https://connect.nuxeo.com/nuxeo/site/marketplace/package/amazon-s3-online-storage?version=11.2.13)
```

Comment:
Guess the `amazon-s3-online-storage` version is not the correct one. That's for 11.x or am I missing something?

Reply:
Guess the most up to date version for 10.10 would be `1.9.12`

Reply:
i just did the link for the connector, I think it must grab the latest by default.  will be sure to grab the specific one for 10. 👍